addons/point_of_sale/static/src/css/pos.css

```diff
@@ -2999,16 +2999,43 @@ td {
 
 .pos .product_configurator_attributes .attribute {
     margin-bottom: 1em;
-    display: inline-block;
-    width: 45%;
     padding-left: 0.5em;
-    vertical-align: top;
 }
 
 @media screen and (max-width: 768px) {
     .pos .product_configurator_attributes .attribute {
         width: 95%;
     }
+
+    .pos .product_configurator_attributes .configurator_radio table {
+        width: 90%;
+    }
+
+    .pos .product_configurator_attributes .configurator_radio .price-extra-cell {
+        min-width: 100px;
+    }
+
+    .pos .product_configurator_attributes .configurator_radio .custom-value-cell {
+        display: none;
+    }
+
+    .pos .product_configurator_attributes .configurator_radio .custom-value-cell-mobile {
+        display: flex !important;
```

Comment:
do we really need to put this "!important" ? Can't we just put the rules after the ones that are not in the media tag, and avoid the need of important. (same for other important)

Reply:
nice tip, I didn't know hat